main/test/ca/uwaterloo/flix/library/TestMutList.flix

```diff
@@ -1200,109 +1207,34 @@ namespace TestMutList {
     @test
     def compress!01(): Bool & Impure =
         let v = MutList.range(0, 100);
-        List.foreach(_ -> let _ = MutList.pop!(v); (), List.range(0, 75));
-        let c = MutList.capacity(v);
+        let f = _ -> { let _ = MutList.pop!(v); () };
+        List.foreach(f, List.range(0, 75));
+        let c = capacity(v);
         let _ = MutList.pop!(v);
-        c == 100 and MutList.capacity(v) == 50
+        capacity(v) == c / 2 and
+        MutList.length(v) == 100 - 75 - 1
```

Comment:
The Flix style guide hasn't been published yet, but IMO we should indent expression continuations like this.

Reply:
Alright, I like that, will do that on future tests I write / refactor.